common/src/main/java/io/apicurio/registry/rest/v2/SearchResource.java

```diff
@@ -29,7 +29,9 @@ ArtifactSearchResults searchArtifacts(@QueryParam("name") String name,
       @QueryParam("offset") Integer offset, @QueryParam("limit") Integer limit,
       @QueryParam("order") SortOrder order, @QueryParam("orderby") SortBy orderby,
       @QueryParam("labels") List<String> labels, @QueryParam("properties") List<String> properties,
-      @QueryParam("description") String description, @QueryParam("group") String group);
+      @QueryParam("description") String description, @QueryParam("group") String group,
+      //TODO get Long from apicurio studio
+      @QueryParam("globalId") Integer globalId, @QueryParam("contentId") Integer contentId);
```

Comment:
so it's not possible to get `Long` from apicurio studio code generator?

Reply:
It should generate `Long` next time we regenerate the JAX-RS interfaces now that I've changed the OpenAPI definition to use `int64` for those properties.  I just didn't bother regenerating it.